core/payment-driver/gnt/src/lib.rs

```diff
@@ -215,43 +233,55 @@ impl GntDriver {
         .unwrap())))
     }
 
+    /// Obtains funds from faucet
+    fn fund<'a>(
+        &self,
+        address: &str,
+    ) -> Pin<Box<dyn Future<Output = GNTDriverResult<String>> + 'a>> {
+        let address = utils::str_to_addr(address).unwrap();
```

Comment:
`.map_err()?` instead of unwrap?

Reply:
Unfortunately, that won't work. :cry: I could only do that *within* the async move block but I need the converted address before to create the futures.

Reply:
I will move the type conversion one level up to remove `unwrap`.